sql/core/src/main/scala/org/apache/spark/sql/execution/adaptive/AdaptiveSparkPlanExec.scala

```diff
@@ -713,10 +735,15 @@ case class AdaptiveSparkPlanExec(
   private def reOptimize(logicalPlan: LogicalPlan): Option[(SparkPlan, LogicalPlan)] = {
     try {
       logicalPlan.invalidateStatsCache()
-      val optimized = optimizer.execute(logicalPlan)
-      val sparkPlan = context.session.sessionState.planner.plan(ReturnAnswer(optimized)).next()
-      val newPlan = applyPhysicalRules(
-        applyQueryPostPlannerStrategyRules(sparkPlan),
+      val optimized = withRuleContext {
+        optimizer.execute(logicalPlan)
+      }
```

Comment:
I'm wondering if we can make it as one-liner? Or, did `scalastyle` for `scalafmt` complain for this one-liner style?
```scala
val optimized = withRuleContext { optimizer.execute(logicalPlan) }
```